mongo/src/main/java/com/redhat/lightblue/mongo/crud/MongoLocking.java

```diff
@@ -80,12 +80,8 @@ private boolean acquire(String callerId,String resourceId, Long ttl, Date now, D
             append(COUNT,1).
             append(VERSION,1);
 
-        try {
-            LOGGER.debug("insert: {}",update);
-            coll.insert(update,WriteConcern.SAFE);
-        } catch (MongoException.DuplicateKey e) {
-            return false;
-        }
+        LOGGER.debug("insert: {}", update);
+        coll.insert(update, WriteConcern.ACKNOWLEDGED);
```

Comment:
Bad change on my part.  Should have substituted for DuplicateKeyException.  Fixing.